packages/console/package.json

```diff
@@ -5,7 +5,9 @@
   "dependencies": {
     "@theia/core": "1.12.0",
     "@theia/monaco": "1.12.0",
-    "anser": "^2.0.1"
+    "@types/dompurify": "^2.0.2",
+    "anser": "^2.0.1",
```

Comment:
If these dependencies/versions are new to the framework and not done already, please recursively check the runtime dependencies for license compatibility.

Reply:
@marcdumais-work they are already present in other extensions:

https://github.com/eclipse-theia/theia/blob/d9c0fabb4e657e7d1879a8067d119598cc00fb43/packages/preview/package.json#L10

https://github.com/eclipse-theia/theia/blob/b0ed2acfff8cf823f3e794bbc5a8dead8cc9c6da/packages/plugin-ext/package.json#L31

I can do a license check anyways :+1: 
We might also want to align versions across the framework.

Reply:
I confirmed that the dependencies are license compatible.